### DISCUSSION TITLE
How create dropdown menu using Vue package?

### DISCUSSION THREAD

**Discussion by lannodev** [floating-ui/floating-ui#2391] [Q&A]
I want to create a dropdown menu like this example for react: https://codesandbox.io/s/admiring-lamport-5wt3yg?file=/src/DropdownMenu.tsx

This is possible using Vue package?
All the best.

**Comment by roma-marshall** [comment]
@luciano-work take a look at this, it might help you: https://vuejsexamples.com/tag/dropdown/
